resources/js/processes/scripts/components/ScriptEditor.vue

```diff
@@ -1,12 +1,7 @@
 <template>
   <b-container class="h-100">
     <b-card no-body class="h-100">
-      <b-card-header class="text-right">
-        <b-button title="Save Script" @click="save" size="sm">
-          <i class="fas fa-save" />
-          {{ $t('Save') }}
-        </b-button>
-      </b-card-header>
+      <top-menu ref="menuScript" :options="optionsMenu"/>
```

Comment:
This ref isn't used anywhere and can be removed.

```suggestion
      <top-menu :options="optionsMenu"/>
```